authenticator/App.js

```diff
@@ -3,19 +3,20 @@ import React from 'react';
 import { NavigationContainer } from '@react-navigation/native';
 import { createNativeStackNavigator } from '@react-navigation/native-stack';
 import { StyleSheet, View } from 'react-native';
+import Toast from 'react-native-toast-message';
 import ApplicationList from './views/application-list';
 import Settings from './views/settings';
 import New from './views/new';
 import Confirm from './views/confirm';
 import License from './views/license';
+import Privacy from './views/privacy-policy';
```

Comment:
I believe there is no such file at the moment. We can probably just paste the html that we had from that generator and format it with similar js as before!